frontend/public/style/_config.scss

```diff
@@ -8,5 +8,5 @@ $fa-font-path: $coreosWebFontPath;
 $icon-font-path: $coreosWebFontPath + '/';
 // PatternFly font path.
 $font-path: '~patternfly/dist/fonts/';
-// CoreOS-web image path.
-$img-path: $coreosWebDistPath + 'img';
```

Comment:
Before we remove this, do we know how this was used?

Reply:
I talked to Jeff about how to do this and he said he didn't see it used anywhere.